docs/pages/reference/glossary.mdx

```diff
@@ -7,18 +7,18 @@ description: Glossary of key Teleport terms, keywords, and definitions.
 
 | Term | Description | Link |
 | --- | --- | --- |
-| <h4 id="auth">Auth</h4> | One of three primary modes that a Teleport Node can be put into. *Teleport Auth Service* uses two Certificate Authorities (CA): one to sign user keys, the other signs Teleport Node keys. | [Teleport Auth](../architecture/authentication.mdx) |
-| <h4 id="teleport-cluster">Cluster</h4> | A collection of Teleport Nodes connected to the same Teleport Certificate Authority is called a *Teleport Cluster*. | [Architecture Overview](../architecture/overview.mdx) |
```

Comment:
I think it's potentially helpful to have these get wrapped in **h4**. Problems:

1. Renders too large.
2. **h3** is way too large.
3. **h5** is styled in full caps - otherwise would be ideal.
4. **h6** is too small.
5. Passing an `id` into a `div` allows the URL to go to the anchor hash but the selector doesn't display for anything but **h** elements it seems.